src/background/providers/cloudflare.test.ts

```diff
@@ -117,11 +120,13 @@ describe('issuance', () => {
             expect(navigateUrl).not.toHaveBeenCalled();
 
             const issueInfo = provider['issueInfo'];
-            expect(issueInfo!.requestId).toEqual(details.requestId);
-            expect(issueInfo!.formData).toStrictEqual({
-                ['h-captcha-response']: 'body-param',
-                ['cf_captcha_kind']: 'body-param',
-            });
+            if (issueInfo) {
```

Comment:
Could this be more explicit whether it's checking for `false`, `null`, or `undefined`?

Reply:
done